packages/block-library/src/post-featured-image/dimension-controls.js

```diff
@@ -134,8 +134,7 @@ const DimensionControls = ( {
 				panelId={ clientId }
 			>
 				<SelectControl
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/caf4fa64-c33f-427f-9abb-bce9a31213f3" alt="Featured Image block inspector, before" width="274">|<img src="https://github.com/user-attachments/assets/bb217cd4-5ae8-45a1-903c-c7dc460387ea" alt="Featured Image block inspector, after" width="274">|

Incorrect column gap will be addressed separately at #64488